examples/Remarketing/AddRealEstateFeed.php

```diff
@@ -0,0 +1,440 @@
+use Google\Ads\GoogleAds\V2\Services\FeedMappingOperation;
+use Google\Ads\GoogleAds\V2\Services\FeedOperation;
+use Google\Ads\GoogleAds\V2\Services\GoogleAdsRow;
+use Google\ApiCore\ApiException;
+use Google\Protobuf\Int64Value;
+use Google\Protobuf\StringValue;
+
+/** Adds a real estate feed, creates the feed mapping, and adds items to the feed. */
+class AddRealEstateFeed
+{
+    const CUSTOMER_ID = 'INSERT_CUSTOMER_ID_HERE';
+    const PAGE_SIZE = 1000;
+
+    public static function main()
+    {
+        // Either pass the required parameters for this example on the command line, or insert them
+        // into the constants above.
+        $options = (new ArgumentParser())->parseCommandArguments([
+            ArgumentNames::CUSTOMER_ID => GetOpt::REQUIRED_ARGUMENT
+        ]);
+
+        // Generate a refreshable OAuth2 credential for authentication.
+        $oAuth2Credential = (new OAuth2TokenBuilder())->fromFile()->build();
+
+        // Construct a Google Ads client configured from a properties file and the
+        // OAuth2 credentials above.
+        $googleAdsClient = (new GoogleAdsClientBuilder())
+            ->fromFile()
+            ->withOAuth2Credential($oAuth2Credential)
+            ->build();
+
+        try {
+            self::runExample(
+                $googleAdsClient,
+                $options[ArgumentNames::CUSTOMER_ID] ?: self::CUSTOMER_ID
+            );
+        } catch (GoogleAdsException $googleAdsException) {
+            printf(
+                "Request with ID '%s' has failed.%sGoogle Ads failure details:%s",
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     */
+    public static function runExample(GoogleAdsClient $googleAdsClient, int $customerId)
+    {
+        // Creates a new feed, but you can fetch and re-use an existing feed by skipping the
+        // createFeed method and inserting the feed resource name of the existing feed into the
+        // getFeed method.
+        $feedResourceName = self::createFeed($googleAdsClient, $customerId);
+        // Gets the page feed details.
+        $placeHoldersToFeedAttributesMap =
```

Comment:
FYI, I'm developing a Util that helps feed-related examples, so we could probably combine them in the future.